components/sections/faqs/variant_b.tsx

```diff
@@ -24,80 +29,98 @@ function VariantB({ subtitle, title, faqsWithCategories }: FAQProps) {
-                <button
-                  aria-label={tab?.category}
-                  key={index}
-                  onClick={() => setTabPane(tab?.category)}
-                  className={
-                    tabPane === tab?.category
-                      ? "w-full border-b-2 border-webriq-blue px-4 py-4 font-bold text-webriq-darkblue md:w-1/2 lg:w-auto"
-                      : "w-full border-b-2 border-transparent px-4 py-4 font-bold text-gray-500 transition duration-150 hover:border-webriq-blue hover:text-webriq-darkblue md:w-1/2 lg:w-auto"
-                  }
-                >
-                  {tab?.category}
-                </button>
-              ))}
-            </div>
-          </div>
-        </div>
-        <div className="container mx-auto px-4">
-          <div className="mx-auto max-w-3xl">
-            <ul>
-              {updatedFAQArray
-                ?.find((item) => item?.category === tabPane)
-                ?.askedQuestions?.map((content, index) => (
-                  <li className="border-b py-12 pr-4" key={index}>
-                    <button
-                      aria-label={content?.question}
-                      className="font-heading flex w-full items-center justify-between text-left font-bold hover:text-gray-600 focus:outline-none"
-                      onClick={() => toggleView(index)}
-                    >
-                      <span className="w-3/4 text-xs lg:text-xl xl:text-xl 2xl:text-xl">
-                        {content?.question}
-                      </span>
-                      <svg
-                        className="h-4 w-4 text-webriq-darkblue lg:h-6 lg:w-6 xl:h-6 xl:w-6 2xl:h-6 2xl:w-6"
-                        xmlns="http://www.w3.org/2000/svg"
-                        fill="none"
-                        viewBox="0 0 24 24"
-                        stroke="currentColor"
-                      >
-                        <path
-                          strokeLinecap="round"
-                          strokeLinejoin="round"
-                          strokeWidth={2}
-                          d={
-                            show && activeQA === index
-                              ? "M5 10l7-7m0 0l7 7m-7-7v18"
-                              : "M19 14l-7 7m0 0l-7-7m7 7V3"
-                          }
-                        />
-                      </svg>
-                    </button>
-                    {show && activeQA === index && (
-                      <p className="mt-4 text-xs font-normal leading-loose text-gray-500 lg:text-xl xl:text-xl 2xl:text-xl">
-                        {content?.answer}
-                      </p>
-                    )}
-                  </li>
-                ))}
-            </ul>
-          </div>
-        </div>
+    <section className="py-20 bg-gray-50">
+      <div className="border-b">
+        <Container maxWidth={576} className="mb-16 text-center ">
+          <Text weight="bold" className="text-primary">
```

Comment:
Kindly update the Subtitle and Title elements to be returned only when it is defined/there is a value.